KerbalStuff/config.py

```diff
@@ -20,6 +21,7 @@ def get_env_var_or_config(section, key):
 _cfg = lambda k: get_env_var_or_config(env, k)
 _cfgi = lambda k: int(_cfg(k))
 _cfgb = lambda k: strtobool(_cfg(k)) == 1
+_cfgl = lambda k: ast.literal_eval(_cfg(k))
```

Comment:
_Note to myself_: at some point I should refactor these horrible lambdas into proper accessor functions with exception handling and all.